sklearn/base.py

```diff
@@ -31,29 +31,34 @@
 
 
 def clone(estimator, *, safe=True):
-    """Constructs a new unfitted estimator with the same parameters.
-
-    Clone does a deep copy of the model in an estimator
-    without actually copying attached data. It yields a new estimator
-    with the same parameters that has not been fitted on any data.
-
-    If the estimator's `random_state` parameter is an integer (or if the
-    estimator doesn't have a `random_state` parameter), an *exact clone* is
-    returned: the clone and the original estimator will give the exact same
-    results. Otherwise, *statistical clone* is returned: the clone might
-    yield different results from the original estimator. More details can be
-    found in :ref:`randomness`.
+    """Construct a new unfitted estimator with the same parameters.
 
     Parameters
     ----------
     estimator : {list, tuple, set} of estimator instance or a single \
             estimator instance
         The estimator or group of estimators to be cloned.
-
     safe : bool, default=True
         If safe is False, clone will fall back to a deep copy on objects
         that are not estimators.
 
+    Returns
+    -------
+    estimator : Estimator instance
+        The deep copy of the estimator.
+
+    Notes
+    -----
+    Clone does a deep copy of the model in an estimator
+    without actually copying attached data. It returns a new estimator
+    with the same parameters that has not been fitted on any data.
```

Comment:
I think this paragraph is very important and should be left as paragraph before the `Parameters` section.

Reply:
Thanks for your feedback! I will move it up :)